pkg/proxy/meta-proxier.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package proxy
+
+import (
+	"k8s.io/api/core/v1"
+	"k8s.io/klog"
+	"net"
+)
+
+type MetaProxier struct {
+	ipv4Proxier   ProxyProvider
+	ipv6Proxier   ProxyProvider
+	defaultFamily IpFamily
+}
+
+type IpFamily int
+
+const (
```

Comment:
Did we have to export these?

Reply:
No, not after the refactoring. I will fix it